test/e2e_env/kubernetes/externalname-services/externalname-services.go

```diff
@@ -0,0 +1,61 @@
+package externalname_services
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	. "github.com/kumahq/kuma/test/framework"
+	"github.com/kumahq/kuma/test/framework/deployments/testserver"
+	"github.com/kumahq/kuma/test/framework/envs/kubernetes"
+)
+
+func ExternalNameServices() {
+	meshName := "externalname-services"
+	namespace := "externalname-services"
+
+	externalNameService := fmt.Sprintf(`
+apiVersion: v1
+kind: Service
+metadata:
+  name: externalname-service
+  namespace: %s
+spec:
+  type: ExternalName
+  externalName: foo.bar
+  ports: 
+  - appProtocol: tcp
+    port: 3000
+    protocol: TCP
+    targetPort: 3000
+  selector:
+    app: test-server
+`, namespace)
+
+	BeforeAll(func() {
+		Expect(NewClusterSetup().
+			Install(MeshKubernetes(meshName)).
+			Install(NamespaceWithSidecarInjection(namespace)).
+			Install(testserver.Install(
+				testserver.WithName("test-server"),
+				testserver.WithNamespace(namespace),
```

Comment:
please deploy this in created mesh. Just in case

Reply:
addressed